### ISSUE TITLE
ngx_stream_lua_api.c is missing?

### ISSUE THREAD

**Issue by antony66** [openresty/stream-lua-nginx-module#37]
File `api/ngx_stream_lua_api.h` defines some functions like `ngx_stream_lua_add_package_preload` and `ngx_stream_lua_get_session` but I am unable to find their source. I can assume `ngx_stream_lua_api.c` is missing after looking at https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_api.c

Am I wrong?

**Comment by agentzh** [comment]
@antony66 Yeah, they might be missing since the existing test suite do not cover these APIs yet. Patches welcome! :)
